src/event_callback.js

```diff
@@ -0,0 +1,46 @@
+import Car from "./car.js";
+import { showElements } from "./elements.js";
+import startRace from "./race.js";
+
+function checkNames(elems) {
+  let result = true;
+  elems.forEach(function (item) {
+    if (item.length > 5) {
+      result = false;
+    }
+  });
```

Comment:
```suggestion
  return !(elems.filter((item) => item.length).length > 0);
```

filter를 사용하면 불필요한 result를 선언 안할 수 있어서 좋은것같아요! 실제 lint에서도 forEach 내부에서 외부 변수를 바꾸면 GC가 추적하기 힘들어서 Error를 내보내더라구요.